x/ibc/light-clients/07-tendermint/client/cli/tx.go

```diff
@@ -119,7 +119,8 @@ func NewCreateClientCmd() *cobra.Command {
 			allowUpdateAfterExpiry, _ := cmd.Flags().GetBool(flagAllowUpdateAfterExpiry)
 			allowUpdateAfterMisbehaviour, _ := cmd.Flags().GetBool(flagAllowUpdateAfterMisbehaviour)
 
-			upgradePath, _ := cmd.Flags().GetString(flagUpgradePath)
+			upgradePathStr, _ := cmd.Flags().GetString(flagUpgradePath)
```

Comment:
the flag set should give you the option to use a string slice instead of a plain string, that will save you the split below and will automatically validate that the input is a comma-separated value

Reply:
I wouldn't spend time on the light client CLI. We should be removing these soon and this command doesn't even work, it is broken in multiple parts